test/testmachinery/shoots/applications/shoot_app.go

```diff
@@ -57,7 +57,7 @@ var _ = ginkgo.Describe("Shoot application testing", func() {
 	f := framework.NewShootFramework(nil)
 
 	f.Default().Release().CIt("should download shoot kubeconfig successfully", func(ctx context.Context) {
-		err := framework.DownloadKubeconfig(ctx, f.SeedClient, f.ShootSeedNamespace(), v1beta1constants.SecretNameGardener, "")
+		err := framework.DownloadKubeconfig(ctx, f.SeedClient, f.ShootSeedNamespace(), v1beta1constants.SecretNameGardener, "test-kubecfg")
```

Comment:
Thanks for catching and fixing! I obviously missed it.

I think we can avoid saving the downloaded kubeconfig to the filesystem:

```suggestion
	f.Default().Release().CIt("should fetch the shoot kubeconfig from the Seed cluster successfully", func(ctx context.Context) {
		kubeconfig, err := framework.GetObjectFromSecret(ctx, f.SeedClient, f.ShootSeedNamespace(), v1beta1constants.SecretNameGardener, framework.KubeconfigSecretKeyName)
		framework.ExpectNoError(err)
		Expect(kubeconfig).NotTo(BeEmpty())

		ginkgo.By("Shoot kubeconfig fetched successfully from the Seed cluster")
	}, downloadKubeconfigTimeout)
```

We are only interested to verify that the Secret exists with non-empty content.